docker-compose.yml

```diff
@@ -3,6 +3,22 @@ version: '3.1'
 
 services:
 
+  localstack:
+    image: localstack/localstack
+    ports:
+      - "53:53"
+      - "443:443"
+      - "4510-4520:4510-4520"
+      - "4566-4620:4566-4620"
+      - "${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}"
+    environment:
+      - DEBUG=1
```

Comment:
Suggestion (blocking): we probably doesn't need `DEBUG`. Also would you set `SERVICES=s3` so that we activate s3 only?

Reply:
Yeah you're right, its faster when we add the ` - SERVICES=s3` to the `docker-compose.yaml`